### DISCUSSION TITLE
Considering cloud pricing models

### DISCUSSION THREAD

**Discussion by dannyshin76** [Budibase/budibase#10643] [Help] [answered]
We are currently using self-hosting environment, and looking to using cloud pricing models (possibly Business tier).
So we have tried free tier cloud first, but the user experience was not exactly comfortable (app loading speed, REST API response time, etc).
Does the Business tier come with a dedicated server resource? Or it just shares the same resource pool with other tiers?
In other words, can we expect performance improvement with Business tier?

**Comment by ConorWebb96** [comment, marked as answer]
Hey @dannyshin76,

The free tier is on the same infrastructure as the paid-for (business) tiers, so there should be no performance differences when you upgrade.

Whilst performance of Budibase is something we’re continually looking at improving, it’s unusual for there to be a noticeable difference between self-host and cloud-host, and many things can affect that performance (app structure, DB used, geographical location due to latency etc.).

I’m happy to set up a call to run through how you’re using Budibase to see if there are any suggestions to help improve performance? Please email [[support@budibase.com](mailto:support@budibase.com)] and I can get that set up.

**Reply by dannyshin76** [reply]
Thanks for information!
